pkgs/by-name/va/vaults/package.nix

```diff
@@ -22,23 +22,25 @@
 
 stdenv.mkDerivation rec {
   pname = "vaults";
-  version = "0.8.0";
+  version = "0.9.0";
 
   src = fetchFromGitHub {
     owner = "mpobaschnig";
-    repo = "Vaults";
-    rev = "v${version}";
-    hash = "sha256-USVP/7TNdpUNx1kDsCReGYIP8gHUeij2dqy8TR4R+CE=";
+    repo = "vaults";
+    tag = version;
+    hash = "sha256-PczDj6G05H6XbkMQBr4e1qgW5s8GswEA9f3BRxsAWv0=";
   };
 
   cargoDeps = rustPlatform.fetchCargoVendor {
     inherit src;
     name = "${pname}-${version}";
-    hash = "sha256-93X2BCn6Ih2DqYJNvYvUCYrC1E6wpCT5X8Hyux/8mno=";
+    hash = "sha256-j0A6HlApV0l7LuB7ISHp+k/bSH5Icdv+aNQ9juCCO9I=";
   };
 
   postPatch = ''
     patchShebangs build-aux
+    substituteInPlace src/global_config_manager.rs \
+      --replace-fail 'expect("Could not load .flatpak-info")' "unwrap_or_else(|_| Ini::new())"
```

Comment:
I am not entirely sure, what the current standard is, but potentially this would be better suited as a patch.